aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/utils/FSUtils.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.xray.utils;
+
+import com.amazonaws.xray.entities.StringValidator;
+import java.nio.file.FileSystem;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class FSUtils {
```

Comment:
Ditto, let's not add any public API we don't need (or we could put in `internal` package if we really wanted to)